AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventManagerInviteController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AllReady.Areas.Admin.Features.EventManagerInvites;
+using AllReady.Areas.Admin.ViewModels.ManagerInvite;
+using AllReady.Features.Events;
+using AllReady.Models;
+using AllReady.Security;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace AllReady.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize(nameof(UserType.OrgAdmin))]
+    public class EventManagerInviteController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly UserManager<ApplicationUser> _userManager;
+
+        public EventManagerInviteController(IMediator mediator, UserManager<ApplicationUser> userManager)
+        {
+            _mediator = mediator;
+            _userManager = userManager;
+        }
+
+        // GET: Admin/Invite/Send/5
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> Send(int eventId)
+        {
+            var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
```

Comment:
just a thought. `EventByEventIdQueryHandler` loads ALL of these entities:
```
return await _context.Events
.Include(a => a.Location)
.Include(a => a.Campaign).ThenInclude(c => c.ManagingOrganization)
.Include(a => a.RequiredSkills).ThenInclude(rs => rs.Skill).ThenInclude(s => s.ParentSkill)
.Include(a => a.VolunteerTasks).ThenInclude(t => t.AssignedVolunteers).ThenInclude(tu => tu.User)
.Include(a => a.VolunteerTasks).ThenInclude(t => t.RequiredSkills).ThenInclude(ts => ts.Skill)
.SingleOrDefaultAsync(a => a.Id == message.EventId);
```

builds and returns an entire `Event`. You need a very small subset of that data, namely, you need just enough to build  your ViewModel 
```
return View("Send", new EventManagerInviteViewModel
{
    EventId = eventId,
    EventName = @event.Name,
    CampaignId = @event.CampaignId,
    CampaignName = @event.Campaign.Name,
});
```

I'd write a new query and handler (maybe name it `EventManagerInviteQuery`) and just query the data you need to create the EventManagerInviteViewModel and return that from the query handler.

FYI, some of out controllers in the solution try to work with a ViewModel per controller (aka, do not share ViewModels, which is a good a practice to follow), or go even further if the controller is large or complicated, where there is a ViewModel per GET/POST action method. 

Check out [SiteAdminController's Index action method](https://github.com/HTBox/allReady/blob/62f5f6bb1edc8d91fde7c2aeb02a663cffcd3b61/AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/SiteAdminController.cs#L44), IndexQuery and IndexQueryHandler.